pkg/api/common/helper_test.go

```diff
@@ -78,39 +46,31 @@ func Test_RationalizeReleaseAndVersion(t *testing.T) {
 		t.Errorf("It is not the default Kubernetes version")
 	}
 
+	latest1Dot6Version := GetLatestPatchVersion("1.6", GetAllSupportedKubernetesVersions())
```

Comment:
We already have conveniences to grab the latest version in a major.minor release channel, so we don't need to be maintaining them statically to validate "orchestratorRelease" behavior.